Common/SymbolJsonConverter.cs

```diff
@@ -72,7 +72,14 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
                 underlyingSymbol = new Symbol(SecurityIdentifier.Parse(underlying["ID"].ToString()), underlying["Value"].ToString());
             }
 
-            return new Symbol(SecurityIdentifier.Parse(jobject["ID"].ToString()), jobject["Value"].ToString(), underlyingSymbol);
+            var symbolId = jobject["ID"];
+
+            if (symbolId == null)
+            {
+                return null;
+            }
+
+            return new Symbol(SecurityIdentifier.Parse(symbolId.ToString()), jobject["Value"].ToString(), underlyingSymbol);
         }
```

Comment:
Just  some  context: the reason we check it for null is backwards compatibility with the previous symbol definition, which didn't have the security identifier property